man/mustache.5.ron

```diff
@@ -411,6 +411,40 @@ Can be thought of as a single, expanded template:
     {{/names}}
 
 
+### Dynamic Names
```

Comment:
The manpage, at least in its current form, is tag-oriented. Dotted names and lambdas are discussed as sub-features of variable tags and block tags. For consistency, I would suggest similarly treating dynamic names as a subsection of partial tags. That means formatting the title in bold, instead of as a level 3 heading:

```suggestion
**Dynamic Names**
```

Conveniently, you already placed this section exactly where it needs to be in order to discuss it as a subfeature of partial tags.

Reply:
Right!